app/components/partner/client-report-download/index.hbs

```diff
@@ -96,7 +99,6 @@
               @clientId={{@clientId}}
               @fileId={{@fileId}}
               @reportId={{this.latestReport.id}}
-              id='password-toggle-{{this.latestReport.id}}'
```

Comment:
why is this removed ?

Reply:
It wasn't removed entirely. it was relocated [here](https://github.com/appknox/irene/pull/1284/files#diff-0b89776586e7aafa9082f0c376c3946bd69a93416c2b764134531acf1eba1d17R13)